Library/Homebrew/extend/os/mac/keg_relocate.rb

```diff
@@ -28,12 +28,14 @@ def relocate_dynamic_linkage(relocation)
 
         each_linkage_for(file, :dynamically_linked_libraries) do |old_name|
           new_name = relocated_name_for(old_name, relocation)
-          change_install_name(old_name, new_name, file) if new_name
+          new_name = loader_name_for(file, new_name)
```

Comment:
Can we also avoid recycling the `new_name` variable? I feel like it makes understanding what's going on a bit more confusing than it should be. (But feel free to leave it as is if you have a strong opinion here.)

Reply:
No strong opinion. I've changed it (but let me know if you think of a better name or names).